javascript/express/security/injection/tainted-sql-string.yaml

```diff
@@ -44,11 +45,12 @@ rules:
               - pattern: $REQ.headers
       - patterns:
           - pattern-either:
-              - pattern-inside: >
+              - pattern-inside: |
                   ({ $REQ }: Request,$RES: Response, $NEXT: NextFunction) =>
                   {...}
               - pattern-inside: |
                   ({ $REQ }: Request,$RES: Response) => {...}
+          - focus-metavariable: $REQ
```

Comment:
Here this is what you want! Because you want `params` etc to refer to values of `$REQ`.